eos_data_distribution/SimpleStore/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 class Producer(object):
 
-    def __init__(self, base=None, prefix='/',
+    def __init__(self, base='.', prefix='/',
```

Comment:
I’m not sure we should be encouraging use of relative paths here — where do we set the `cwd`? I think it would be more explicit to require that `base` be passed in. The caller can always pass in `'.'` if it’s set up the `cwd` appropriately.